tests/network/macvtap.go

```diff
@@ -87,184 +78,21 @@ var _ = SIGDescribe("Macvtap", decorators.Macvtap, Serial, func() {
 			chosenMACHW, err := GenerateRandomMac()
 			Expect(err).ToNot(HaveOccurred())
 			chosenMAC = chosenMACHW.String()
-			serverCIDR := "192.0.2.102/24"
 
-			serverIP, err = libnet.CidrToIP(serverCIDR)
+			const macvtapNetName = "test-macvtap"
+			serverVMI := libvmi.NewAlpineWithTestTooling(
+				libvmi.WithInterface(*libvmi.InterfaceWithMac(v1.DefaultMacvtapNetworkInterface(macvtapNetName), chosenMAC)),
+				libvmi.WithNetwork(libvmi.MultusNetwork(macvtapNetName, macvtapNetworkName)),
+				libvmi.WithNodeAffinityFor(nodeName),
+			)
+			serverVMI, err = kubevirt.Client().VirtualMachineInstance(testsuite.GetTestNamespace(serverVMI)).Create(context.Background(), serverVMI)
 			Expect(err).ToNot(HaveOccurred())
-
-			serverVMI = createAlpineVMIStaticIPOnNode(nodeName, macvtapNetworkName, "eth0", serverCIDR, &chosenMAC)
+			serverVMI = libwait.WaitUntilVMIReady(serverVMI, console.LoginToAlpine)
 		})
 
 		It("should have the specified MAC address reported back via the API", func() {
```

Comment:
This scenario should be covered in the plugin test as well.

Reply:
Done